packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -425,7 +415,6 @@ class CreateCommand extends FlutterCommand with CreateBase {
       withFfiPackage: generateFfiPackage,
       withEmptyMain: emptyArgument,
       androidLanguage: stringArg('android-language'),
-      iosLanguage: stringArg('ios-language'),
```

Comment:
The tests are failing because, while I said to delete this line, I totally forgot that this also requires that the `ios-swift` template files that generate new projects be renamed `ios`. I did this for apps here:
https://github.com/flutter/flutter/pull/169547
Example: 
https://github.com/flutter/flutter/tree/09d4dabd6d69fc10855c7b0c9c996f337adf76b6/packages/flutter_tools/templates/plugin/ios-swift.tmpl
https://github.com/flutter/flutter/tree/5a63b1df955bba1939bd7eab2bced5fe016ca429/packages/flutter_tools/templates/plugin_cocoapods/ios-swift.tmpl
https://github.com/flutter/flutter/tree/b98ffdc46761d3f45f95bfc1cb66c6872e9328d6/packages/flutter_tools/templates/plugin_swift_package_manager/ios-swift.tmpl
and anywhere else.